apis/storage/v1alpha1/zz_generated.claim.go

```diff
@@ -60,27 +49,22 @@ func (cm *Bucket) SetBindingPhase(p runtimev1alpha1.BindingPhase) {
 	cm.Status.SetBindingPhase(p)
 }
 
-// SetClassReference of this Bucket.
-func (cm *Bucket) SetClassReference(r *corev1.ObjectReference) {
-	cm.Spec.ClassReference = r
-}
-
-// SetClassSelector of this Bucket.
-func (cm *Bucket) SetClassSelector(s *metav1.LabelSelector) {
-	cm.Spec.ClassSelector = s
-}
-
 // SetConditions of this Bucket.
 func (cm *Bucket) SetConditions(c ...runtimev1alpha1.Condition) {
 	cm.Status.SetConditions(c...)
 }
 
+// SetPortableClassReference of this Bucket.
+func (cm *Bucket) SetPortableClassReference(r *corev1.LocalObjectReference) {
+	cm.Spec.PortableClassReference = r
+}
+
 // SetResourceReference of this Bucket.
 func (cm *Bucket) SetResourceReference(r *corev1.ObjectReference) {
 	cm.Spec.ResourceReference = r
 }
 
 // SetWriteConnectionSecretToReference of this Bucket.
-func (cm *Bucket) SetWriteConnectionSecretToReference(r *runtimev1alpha1.LocalSecretReference) {
+func (cm *Bucket) SetWriteConnectionSecretToReference(r corev1.LocalObjectReference) {
 	cm.Spec.WriteConnectionSecretToReference = r
```

Comment:
@negz I only added this diff because `make reviewable` pulled my arm.  Can #962 include an update to crossplane-runtime v0.2.0 and include an updated `make manifests` (I also found a `go mod tidy` was needed)?